SimpleTerminologyLoadTest.ipynb

```diff
@@ -0,0 +1,331 @@
+      "9.855390071868896\n"
+     ]
+    }
+   ],
+   "source": [
+    "BASE_URL = \"http://20.119.216.32:8000/r4/\"  # JHU Azure TIMS server\n",
+    "import time\n",
+    "from imports import TimsClient\n",
+    "client = TimsClient(BASE_URL, False, False)\n",
+    "\n",
+    "start = time.time()\n",
+    "client.summarize_code_systems()\n",
+    "end = time.time()\n",
+    "print(end - start)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ad5dd31c",
+   "metadata": {},
+   "source": [
+    "Details of a single code system:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "id": "61627370",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "LOINC?\n",
+      "GETTING\n",
+      "http://localhost:8088/fhir/CodeSystem?name=LOINC\n",
+      "SENDING\n",
+      "HTTP STATUS OK 200\n",
+      "name: LOINC\n",
+      "title: Logical Observation Identifiers, Names and Codes (LOINC)\n",
+      "resourceType:  CodeSystem  id: v3-loinc\n",
+      "url: http://loinc.org\n",
+      "full url: http://localhost:8088/fhir/CodeSystem/v3-loinc\n",
+      "identifier system: \"urn:ietf:rfc:3986\",  value: \"urn:oid:2.16.840.1.113883.6.1\"\n",
+      "\n",
+      "SNOMED?\n",
+      "GETTING\n",
+      "http://localhost:8088/fhir/CodeSystem?name=SNOMED+CT\n",
+      "SENDING\n",
+      "HTTP STATUS OK 200\n",
+      "Not Present\n",
+      "\n",
+      "ICD-10?\n",
+      "GETTING\n",
+      "http://localhost:8088/fhir/CodeSystem?name=ICD-10-CM\n",
+      "SENDING\n",
+      "HTTP STATUS OK 200\n",
+      "name: ICD-10-CM\n",
+      "resourceType:  CodeSystem  id: icd10CM\n",
+      "url: http://hl7.org/fhir/sid/icd-10-cm\n",
+      "full url: http://localhost:8088/fhir/CodeSystem/icd10CM\n",
+      "\n",
+      "HPO?\n",
+      "GETTING\n",
+      "http://localhost:8088/fhir/CodeSystem?name=HPO\n",
+      "SENDING\n",
+      "HTTP STATUS OK 200\n",
+      "name: HPO\n",
+      "title: Human Phenotype Ontology\n",
+      "resourceType:  CodeSystem  id: HPO\n",
+      "url: http://human-phenotype-ontology.org\n",
+      "full url: http://localhost:8088/fhir/CodeSystem/HPO\n",
+      "\n",
+      "CPT4?\n",
+      "GETTING\n",
+      "http://localhost:8088/fhir/CodeSystem?name=CPT4\n",
+      "SENDING\n",
+      "HTTP STATUS OK 200\n",
+      "Not Present\n",
+      "\n",
+      "RxNorm\n",
+      "GETTING\n",
+      "http://localhost:8088/fhir/CodeSystem?name=RxNorm\n",
+      "SENDING\n",
+      "HTTP STATUS OK 200\n",
+      "name: RxNorm\n",
+      "title: RxNorm\n",
+      "resourceType:  CodeSystem  id: v3-rxNorm\n",
+      "url: http://www.nlm.nih.gov/research/umls/rxnorm\n",
+      "full url: http://localhost:8088/fhir/CodeSystem/v3-rxNorm\n",
+      "identifier system: \"urn:ietf:rfc:3986\",  value: \"urn:oid:2.16.840.1.113883.6.88\"\n",
+      "1.1222667694091797\n"
+     ]
+    }
+   ],
+   "source": [
```

Comment:
https://quarto.org/
I saw someone link this; mentioned it on Slack the other day and just wanting you to see it. Not suggesting that we change from jupyter notebook for this, but this is basically a variation of jupyter notebook that works better w/ version control. I think it's free and open source. I plan to give it a go at some point.

Reply:
yeah, ipynb including the run output makes for noisy diffs. I do think it's really cool, just trying to ignore it to maintain focus (Squirrel!) on January.